Releases.md

```diff
@@ -4,6 +4,7 @@ See [DirectML version history on MSDN](https://docs.microsoft.com/windows/win32/
 
 | Version                  | Feature level                                                                                                            | First available in OS                                             | Redistributable                                                                             |
 | ------------------------ | ------------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------- | ------------------------------------------------------------------------------------------- |
+| [1.15.0](#directml-1150) | [DML_FEATURE_LEVEL_6_4](https://learn.microsoft.com/windows/ai/directml/dml-feature-level-history#dml_feature_level_6_4) | TBD                                                               | [Microsoft.AI.DirectML.1.15.0](https://www.nuget.org/packages/Microsoft.AI.DirectML/1.15.0) |
```

Comment:
A row was added, but the actual release notes below are missing? `# DirectML 1.15.0 - Foo blah...` (rest looks good)

Reply:
had not saved.. updated.